test/WebJobs.Script.Tests.Integration/packages.config

```diff
@@ -88,6 +89,7 @@
   <package id="System.Dynamic.Runtime" version="4.3.0" targetFramework="net46" />
   <package id="System.Globalization" version="4.3.0" targetFramework="net46" />
   <package id="System.Globalization.Calendars" version="4.3.0" targetFramework="net46" />
+  <package id="System.IdentityModel.Tokens.Jwt" version="5.1.3" targetFramework="net46" />

```

Comment:
To be clear, the failure to build was unrelated (I'd missed updating the packages for this project somehow). But the reinstall added this, which looks correct. It may be b/c we use project references for the tests and they don't auto-pull dependencies when they get added to the source projects.